src/site/content/en/live/index.njk

```diff
@@ -55,7 +55,7 @@ draft: true
 
   <hr />
 
-  <section class="w-event-section">
+  <section id="schedule" class="w-event-section">
```

Comment:
Browsing to https://web.dev/live/#schedule in a new tab results in:

![](https://user-images.githubusercontent.com/81942/82799960-26f55600-9e7b-11ea-8c33-098643c9a285.png)

Could we make it behave more like this?

![](https://user-images.githubusercontent.com/81942/82799994-36749f00-9e7b-11ea-810b-5811ccc9909b.png)

(Asking because I'm using this for a demo and have to currently manually fix the scroll position)